go.mod

```diff
@@ -30,6 +30,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.17.2
 )
 
+require sigs.k8s.io/yaml v1.4.0
```

Comment:
Can this be moved to the already existing `require` block

Reply:
Done